CMake/FindCubeb.cmake

```diff
@@ -0,0 +1,12 @@
+find_path(CUBEB_INCLUDE_DIR cubeb.h PATH_SUFFIXES cubeb)
+
+find_library(CUBEB_LIBRARY cubeb)
+
+set(CUBEB_INCLUDE_DIRS ${CUBEB_INCLUDE_DIR})
```

Comment:
This find module should introduce a proper linkage target like how the original cubeb CMake files do. This way, the `include_directories()` call in the main CMakeLists file becomes unnecessary (and doesn't pollute libraries that don't use cubeb with its includes).